### PR TITLE
Add tests for github webhook callback

### PR DESCRIPTION
**Pull request by jmhooper**
This commit adds tests for the github webhook callback.
This callback is expected to trigger a new build if the request is signed properly and there a new commits.

### CODE REVIEW

File: test/api/requests/webhook.test.js [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+          .expect(200)
+      }).then(response => {
+        return Build.find({ site: site.id, user: user.id })
+      }).then(builds => {
+        expect(builds).to.have.length(2)
+        done()
+      })
+    })
+
+    it("should not schedule a build if there a no new commits", done => {
```
Comment (el-mapache) [inline test/api/requests/webhook.test.js]: I think 'a' should be 'are' here

**Comment by jmhooper** [comment]
Pushed an amended commit to fix the type

**Comment by brandocalrissian** [comment]
Not seeing the amended commit typo fix. Other than that, looks good to go.

**Comment by jmhooper** [comment]
Maybe.... I didn't push an amended commit. Once sec.

**Comment by jmhooper** [comment]
Okay, now it's up there. I'll merge once CI does its thing.

**Comment by brandocalrissian** [comment]
got to be fast around here sometimes...

**Pull request merged**